server/db/db.js

```diff
@@ -15,8 +15,7 @@ const pool = new Pool({
 })
 
 pool.on('error', err => {
-  // eslint-disable-next-line no-console
-  console.error('unexpected database error:', err)
+  helpers.logError('unexpected database error:', err)
```

Comment:
Hmmm, actually now that I'm looking at this again, I think that the `logError` function only takes one parameter. So I probably need to make this all a single string. This is a reminder for me to check this before merging.

Reply:
Yup, the function only takes in one argument, the string to output. So I've changed this now to do that.